Files/BaseLayout.cs

```diff
@@ -256,6 +292,28 @@ public void RightClickContextMenu_Opening(object sender, object e)
             App.InteractionViewModel.CheckForImage();
         }
 
+        private void OpenRecycleBinRightClickContextMenu()
+        {
+            // Remove everything except "Delete", "Cut", "Properties"
+            UnloadMenuFlyoutItemByName("UnzipItem");
+            UnloadMenuFlyoutItemByName("OpenItem");
+            UnloadMenuFlyoutItemByName("OpenInNewTab");
+            UnloadMenuFlyoutItemByName("OpenInNewWindowItem");
+            UnloadMenuFlyoutItemByName("ShareItem");
+            UnloadMenuFlyoutItemByName("CopyItem");
+            UnloadMenuFlyoutItemByName("RenameItem");
+            UnloadMenuFlyoutItemByName("SidebarPinItem");
+            UnloadMenuFlyoutItemByName("QuickLook");
```

Comment:
Being that we are working on adding support for more menu items, it might be a better idea to unload everything through a loop rather than removing the known items, this way we can add more items without updating this list.

Reply:
You mean first removing everything and than adding just what's needed for the recycle bin? Yup I can do that

Reply:
Correct, instead of removing specific ones, remove everything and then add the one we need.

Reply:
I did it a little differently, let me know what you think: I simply created a specialized context menu for recycle bin items so there is no need to Hide/Show menu items and there is no issue when more menu items are added to the "standard" menu.

Reply:
Doing it as an individual context menu is a great idea.